src/main/java/com/palantir/docker/compose/DockerComposeRule.java

```diff
@@ -70,6 +79,11 @@ public DockerComposeExecutable executable() {
     }
 
     @Value.Default
+    public Docker docker() {
+        return new Docker(DockerExecutable.builder().dockerConfiguration(machine()).build());
```

Comment:
There's quite a lot going on in this line which makes it pretty hard to understand. How about making it more like dockerCompose()? This would also address the executable() method now being ambiguous, there could be dockerComposeExecutable() and dockerExecutable() methods.